src/current/_includes/v22.2/ui/statements-filter.md

```diff
@@ -10,14 +10,21 @@
 
 The statement fingerprints returned are determined by the selected **Search Criteria**:
 
-1. By default, the **Top** `100` statement fingerprints **By** `% of All Runtime` are returned.
-  - To change the number returned, select `25`, `50`, or `500` from the dropdown.
-  - To change the column filter, select `Contention Time`, `Execution Count`, `P99 Latency`, or `Statement Time` from the dropdown.
+### Search Criteria ###
+
+By default, the **Top** `100` statement fingerprints **By** `% of All Runtime` for the `Past Hour` are returned.
+
+1. To change the top N number returned, select `25`, `50`, `100`, or `500` from the dropdown.
+1. To change the sort column, from the dropdown, select a commonly sorted column: `% of All Runtime`, `Contention Time`, `Execution Count`, or `Statement Time`.
 1. Select the [**Time Range**](#time-interval).
 1. Click **Apply**.
 
     The list of statements that satisfy the search criteria is displayed. The results can be further [searched and filtered](#search-and-filter).
 
+The actual time range of the statistics is displayed at the top right of the results, for example, `Showing aggregated stats from 20:00 to 21:59 (UTC)`. If you select a preset time interval (not a custom time interval), be aware that the statistics displayed are not automatically refreshed and may become stale. To see the latest statistics, update the **Search Criteria** and apply.
+
+The displayed aggregated statistics may not include data from the last 10 minutes (the default for the [`sql.stats.flush.interval` cluster setting](#statement-statistics)). To reduce page loading time, the results use persisted statistics. For in-memory statistics, directly query the following in-memory table: `crdb_internal.cluster_statement_statistics`.{% if page.cloud != true %} Alternatively, use the [**Active Executions** view](#active-executions-view).{% endif %}
```

Comment:
Style suggestion: Instead of using a parenthentical in the first sentence, you could replace with "... 10 minutes, which is the default..."